src/scores.ts

```diff
@@ -62,127 +67,135 @@ export async function getScores(
   }
 }
 
+async function updateVotesVp(votes: any[], vpState: string, proposalId: string) {
+  const votesWithChange = votes.filter(vote => {
+    const key1 = sha256(JSON.stringify([vote.balance, vote.scores, vpState]));
+    const key2 = sha256(JSON.stringify([vote.vp, vote.vp_by_strategy, vote.vp_state]));
+    return key1 !== key2;
```

Comment:
Why don't we just compare strings directly? we are not reusing these keys, right? I feel it is better to compare strings directly

Reply:
They are not only string, scores by strategy is an array